examples/gcp-values.yaml

```diff
@@ -6,3 +6,53 @@ pachd:
     google:
```

Comment:
Maybe create a separate gcp-values-tls.yaml to ensure we test TLS and non TLS codepaths?